src/jenkins.py

```diff
@@ -610,7 +597,34 @@ def get_updatable_version(proxy: state.ProxyConfig | None = None) -> str | None:
     return latest_version
 
 
-def download_stable_war(container: ops.Container, version: str) -> None:
+def has_lts_updates(proxy: state.ProxyConfig | None = None) -> bool:
+    """Return whether the Jenkins has a patched LTS update available.
+
+    Args:
+        proxy: Proxy server to route the requests through.
+
+    Raises:
+        JenkinsUpdateError: If there was an error fetching the Jenkins version information.
+
+    Returns:
+        True if an update within the same LTS is available. False otherwise.
+    """
+    try:
+        current_version = get_version()
+    except JenkinsError as exc:
```

Comment:
I think you can merge this catch with the one on line 620 if you raise an exception[here](https://github.com/canonical/jenkins-k8s-operator/blob/d85ca11c8a48097bc409975ecf083c23c477d9df/src/jenkins.py#L555). This will potentially lead to a simpler error flow

Reply:
Merged and updated the error message, thank you for the review!